readme.md

```diff
@@ -48,51 +48,92 @@ NB: you can also download MeiliSearch from **Homebrew** or **APT**.
 #### Add documents <!-- omit in toc -->
 
 ```go
-var client = NewClient(Config{
-    Host: "http://127.0.0.1:7700",
-    APIKey: "masterKey"
-})
+package main
 
-resp, err := client.Indexes().Create(CreateIndexRequest{
-    UID: "books",
-})
+import (
+    "fmt"
+    "os"
 
-if err != nil {
-    t.Fatal(err)
-}
+    "github.com/meilisearch/meilisearch-go"
+)
 
-documents := map[int]string{
-    { book_id: 123,  title: "Pride and Prejudice" },
-    { book_id: 456,  title: "Le Petit Prince" },
-    { book_id: 1,    title: "Alice In Wonderland" },
-    { book_id: 1344, title: "The Hobbit" },
-    { book_id: 4,    title: "Harry Potter and the Half-Blood Prince" },
-    { book_id: 42,   title: "The Hitchhiker\'s Guide to the Galaxy" }
+type Book struct {
+    BookID int `json:"book_id"`
+    Title  string `json:"title"`
 }
 
-updateIDRes, err := client.Documents("books").AddOrUpdate(documents) // => { "updateId": 0 }
+func main() {
+    var client = meilisearch.NewClient(meilisearch.Config{
+        Host: "http://127.0.0.1:7700",
+        APIKey: "masterKey"
+    })
+
+    // Create an index if your index does not already exist
+    _, err := client.Indexes().Create(meilisearch.CreateIndexRequest{
+        UID: "books",
+    })
+    if err != nil {
+        fmt.Println(err)
+        os.Exit(1)
+    }
+
+    documents := []Book{
```

Comment:
Instead of creating your struct Book, you can use the JSON type: `map[string]interface{}`. It's just a proposition

Reply:
If I try that:
```go
documents := map[string]interface{}{
		{BookID: 123, Title: "Pride and Prejudice"},
		{BookID: 456, Title: "Le Petit Prince"},
		{BookID: 1, Title: "Alice In Wonderland"},
		{BookID: 1344, Title: "The Hobbit"},
		{BookID: 4, Title: "Harry Potter and the Half-Blood Prince"},
		{BookID: 42, Title: "The Hitchhiker's Guide to the Galaxy"},
	}

updateRes, err := client.Documents("indexUID").AddOrUpdate(documents) // => { "updateId": 0 }
if err != nil {
        fmt.Println(err)
	os.Exit(1)
}

fmt.Println(updateRes.UpdateID)
```

I get:
```bash
./test.go:38:3: missing type in composite literal
```

Reply:
I was thinking about that.

```go
documents := []map[string]interface{}{
		{BookID: 123, Title: "Pride and Prejudice"},
		{BookID: 456, Title: "Le Petit Prince"},
		{BookID: 1, Title: "Alice In Wonderland"},
		{BookID: 1344, Title: "The Hobbit"},
		{BookID: 4, Title: "Harry Potter and the Half-Blood Prince"},
		{BookID: 42, Title: "The Hitchhiker's Guide to the Galaxy"},
	}

updateRes, err := client.Documents("indexUID").AddOrUpdate(documents) // => { "updateId": 0 }
if err != nil {
        fmt.Println(err)
	os.Exit(1)
}

fmt.Println(updateRes.UpdateID)
```

Reply:
Thanks! :)
(I had to chage `BookID` into `"book_id"` and `Title` into `"title"` as well)